heareval/predictions/runner.py

```diff
@@ -107,7 +129,7 @@ def runner(
         )
```

Comment:
Why aren't we passing in the logger here?

Reply:
we get the logger with the help of the task name